src/components/commons/Button.tsx

```diff
@@ -0,0 +1,59 @@
+type ButtonSize = "small" | "medium" | "big";
+
+interface ButtonProps {
+  label: string;
+  href: string;
+  size?: ButtonSize;
+  onClick?: () => void;
+  disabled?: boolean;
+  customStyle?: string;
+}
+
+const getButtonPadding = (size: ButtonSize): string => {
+  switch (size) {
+    case "small":
+      return "py-2 px-5";
+    case "big":
+      return "py-4 px-8";
+    case "medium":
+    default:
+      return "py-3 px-8";
+  }
+};
+
+export default function Button({
+  size = "medium",
+  label,
+  href,
+  onClick,
+  disabled = false,
+  customStyle,
+}: ButtonProps): JSX.Element {
+  const btnPadding = getButtonPadding(size);
+
+  const handleButtonClick = () => {
+    if (onClick) {
+      onClick();
+      return;
+    }
+    window.open(href, "_blank", "noreferrer");
+  };
+
+  return (
+    <button
+      type="button"
+      className={`flex items-center rounded-[28px] bg-black-900 group border border-white-50
+                  hover:brand-bg-gradient-1 hover:border-transparent
+                  ${btnPadding} ${customStyle ?? ""}`}
+      disabled={disabled}
+      onClick={handleButtonClick}
+    >
+      <span
+        data-label={label}
```

Comment:
are you using this attribute anywhere?

Reply:
Nope, remnant of copy-paste 😓  will remove